timelock-server/build.gradle

```diff
@@ -43,11 +45,32 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-core'
     testCompile project(":atlasdb-tests-shared")
 
-    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
 }
 
 distribution {
     serviceName "timelock-server"
     mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
     args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-javaagent:service/javaAgents/jetty-alpn-agent.jar"
```

Comment:
We probably want to specify a default min & max heap size for timelock. I would suggest we start with `-Xms2g -Xmx2g` based on test runs that implicitly selected `-XX:InitialHeapSize=120634624 -XX:MaxHeapSize=1930153984`.